### DISCUSSION TITLE
element-plus 源码打包 错误

### DISCUSSION THREAD

**Discussion by lixiaofa** [element-plus/element-plus#12911] [Q&A]
element-plus 源码打包 错误：
![image](https://github.com/element-plus/element-plus/assets/23230215/c44b02de-fa90-4d72-82bd-c83801a41213)
发现是
<script lang="ts" setup>
import 'vant/es/popup/style'
</script>
这句代码造成的错误

**Comment by xxholly32** [comment]
你用了vant，element-plus里面是不会带vant的

**Reply by lixiaofa** [reply]
> 你用了vant，element-plus里面是不会带vant的

我装了vant ，想也能打包正常 ， 怎么解决大佬

 
![image](https://github.com/element-plus/element-plus/assets/23230215/437d3893-d18b-4849-b397-8b637fa288e3)
他实际是.mjs 文件

**Reply by xxholly32** [reply]
所以这个就不是element-plus的问题了

**Reply by lixiaofa** [reply]
> 所以这个就不是element-plus的问题了

rollup 的问题 ， 但是我不知道问题出在哪

![image](https://github.com/element-plus/element-plus/assets/23230215/47defa0a-cacc-44c8-a604-eaf667d322d4)

**Reply by lixiaofa** [reply]
> 所以这个就不是element-plus的问题了

能帮帮我吗
大佬
